docs/user/integration/aws-cloudwatch/README.md

```diff
@@ -0,0 +1,175 @@
+# Integrate Kyma with AWS CloudWatch
+
+## Overview
+
+The Kyma Telemetry module supports you in integrating with observability backends in a convenient way. The following example outlines how to integrate with [AWS CloudWatch](https://aws.amazon.com/cloudwatch) as a backend. Because CloudWatch does not support OTLP ingestion natively, you must deploy the [AWS Distro for OpenTelemetry](https://aws-otel.github.io) additionally.
+
+![overview](../assets/cloudwatch.drawio.svg)
+
+## Table of Content
+
+- [Prerequisites](#prerequisites)
+- [Preparation](#preparation)
+- [Set Up AWS Credentials](#set-up-aws-credentials)
+- [Deploy the AWS Distro](#deploy-the-aws-distro)
+- [Set Up Kyma Pipelines](#set-up-kyma-pipelines)
+- [Verify the Results](#verify-the-results)
+
+## Prerequisites
+
+- Kyma as the target deployment environment
+- The [Telemetry module](https://kyma-project.io/#/telemetry-manager/user/README) is [enabled](https://kyma-project.io/#/02-get-started/01-quick-install)
+- Kubectl version 1.22.x or higher
+- AWS account with permissions to create new users and security policies
+
+
+## Preparation
+
+1. Export your Namespace as a variable. Replace the `{NAMESPACE}` placeholder in the following command and run it:
+
+    ```bash
+    export K8S_NAMESPACE="{NAMESPACE}"
+    ```
+1. If you haven't created a Namespace yet, do it now:
+    ```bash
+    kubectl create namespace $K8S_NAMESPACE
+    ```
+## Set Up AWS Credentials
+
+### Create AWS IAM User
+
+Create an IAM user and assign to it the specific IAM policies that are needed to let the AWS Distro communicate with the AWS services.
+
+First, create the IAM policy for **CloudWatch** service:
```

Comment:
Isn't there any AWS documentation how to create that policy? This is documenting AWS features, not Kyma features. I think we should only document our own software, not other products. We should simply link to their documentation, not create it among our own docs.

I'd suggest to refrain from click-level (!) documentation of other products and stick to a shorter version. The whole section of lines 41-66 should be summarized as:

1. In your AWS account, create the IAM policies for the following services:
   - **CloudWatch** with the actions **GetMetricData**, **PutMetricData**, and **ListMetrics**.
   - **CloudWatch Logs** with the actions **CreateLogGroup**, **CreateLogStream**, **PutLogEvents**, and **PutRetentionPolicy**, and specify the resource [ARNs](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html) for the selected actions.
2. In your AWS account, create an IAM user and attach the two policies you just created, as well as the policy **AWSXrayWriteOnlyAccess**.
3. In your AWS account, create an access key for an application running outside AWS.